Update environment to have minimal dependencies.

deeplycloudy opened the pull request.

djhoese commented on environment.yml:
```diff
@@ -6,13 +6,14 @@ dependencies:
-- six
\ No newline at end of file
+- scipy
+- six
```
Is `six` needed? This is a python 3-only package, right?

dopplershift replied:
More importantly, it's not a dependency of this code base, so IMO it shouldn't be listed. Let the other packages provide their dependencies.

deeplycloudy replied:
Just pushed a commit to remove six, as well as pyproj, per the principles articulated above.

deeplycloudy commented:
@djhoese It’s needed for lmatools, which still uses six. I haven’t gone through to update lmatools to 3 only.


> On Oct 31, 2019, at 09:23, David Hoese <notifications@github.com> wrote:
> 
> ﻿
> @djhoese commented on this pull request.
> 
> In environment.yml:
> 
> > \ No newline at end of file
> +- scipy
> +- six
> Is six needed? This is a python 3-only package, right?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

deeplycloudy commented:
@dopplershit would I do that with a requirements.txt in lmatools?

> On Nov 1, 2019, at 10:46, Ryan May <notifications@github.com> wrote:
> 
> ﻿
> @dopplershift commented on this pull request.
> 
> In environment.yml:
> 
> > \ No newline at end of file
> +- scipy
> +- six
> More importantly, it's not a dependency of this code base, so IMO it shouldn't be listed. Let the other packages provide their dependencies.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

dopplershift commented:
It’d be in `setup.py` or `setup.cfg` in lmatools.

djhoese commented:
See https://github.com/deeplycloudy/lmatools/pull/26 for python 3-ification of lmatools.

deeplycloudy commented:
@djhoese following the recommendation of @dopplershift I updated lmatools to list its requirements, and added that to your branch that removes python 2 support from lmatools. So, once deeplycloudy/lmatools#26 is merged this PR LGTM.

deeplycloudy commented:
@djhoese after today's changes do you need to do any further testing with this branch before I merge into ugf-newgrid?
